nitransforms/io/itk.py

```diff
@@ -129,10 +129,15 @@ def from_matlab_dict(cls, mdict, index=0):
         tf = cls()
         sa = tf.structarr
 
+        affine = mdict.get(
+            "AffineTransform_double_3_3",
+            mdict.get("AffineTransform_float_3_3")
+        )
```

Comment:
Is one or the other guaranteed to exist at this point? Or should we check `if affine is None` after?

Reply:
Theoretically, ITK allows you any combination of "Translation", "Euler", "Versors", "Rigid", "Similarity" and "Affine" with the data type string and the dimensions numbers.

However, I don't think ANTs produces anything other than Affine.

I'll raise though, to keep the user better informed.